recipes/ucsc-bed-tools/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+mkdir -p "$PREFIX/bin"
+
+if [[ ${target_platform} =~ linux.* ]]; then
+    SRC="rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/"
```

Comment:
This dir should be versioned. Seems to contain version 459 already: https://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/FOOTER.txt

Why you want to use these binaries at all instead of compiling as before: 
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/ucsc-bedgraphtobigwig/build.sh